### ISSUE TITLE
CupertinoAlertDialog Incorrect Text rendering on Flutter 3.22

### ISSUE THREAD

**Issue by Hackmodford** [flutter/flutter#151908]
### Steps to reproduce

Show a Cupertino Dialog in a MaterialApp

### Expected results

<img width="318" alt="IMG_4324" src="https://github.com/user-attachments/assets/46ee6049-bebc-4f47-8258-64c5506e0ef9">

This result was using 3.19.1

### Actual results

<img width="318" alt="IMG_6710" src="https://github.com/user-attachments/assets/d92b514f-d18a-43a1-bc42-f18c6652ec6d">

This result was using 3.22.2

### Code sample

<details open><summary>Code sample</summary>

```dart
showCupertinoDialog(
        context: context,
        builder: (context) {
          return CupertinoAlertDialog(
            title: const Text('A iOS Style Dialog'),
            content: const Text('A lot of content goes here and sometimes the text is fuzzy.'),
            actions: [
              CupertinoDialogAction(
                child: const Text(
                  'OK',
                ),
                onPressed: () {
                  Navigator.pop(context);
                },
              )
            ],
          );
        });
```

</details>

### Screenshots or Video

<details open>
<summary>Screenshots / Video demonstration</summary>

[Upload media here]

</details>

### Logs

<details open><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

### Flutter Doctor output

<details open><summary>Doctor output</summary>

```console
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.22.2, on macOS 14.5 23F79 darwin-arm64, locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 15.4)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2024.1)
[✓] IntelliJ IDEA Ultimate Edition (version 2024.1.3)
[✓] VS Code (version 1.91.1)
[✓] Connected device (6 available)
    ! Error: Browsing on the local area network for Nate's iPhone. Ensure the device is unlocked and attached with a cable or associated with the same local area
      network as this Mac.
      The device must be opted into Developer Mode to connect wirelessly. (code -27)
[✓] Network resources

• No issues found!

```

</details>

**Comment by Hackmodford** [comment]
This seems to be fixed in beta `3.23.0-0.1.pre`

**Comment by danagbemava-nc** [comment]
Hi @Hackmodford, 3.22.3 was released about a day ago, can you check if this reproduces on that version for you? I can't seem to reproduce it on 3.22.3. I added more text to your dialog. See the screenshot below 

<img src="https://github.com/user-attachments/assets/0448288f-ae89-4eb6-bb9e-02e106e20508" width="300"/>

**Comment by Hackmodford** [comment]
@danagbemava-nc It is fixed in 3.22.3

**Comment by danagbemava-nc** [comment]
Thanks for confirming. Closing this as fixed
